tensorflow/c/experimental/stream_executor/stream_executor.h

```diff
@@ -428,9 +428,6 @@ typedef struct SP_Platform {
   // capital letters and underscores.
   const char* type;
 
-  // Number of visible devices
```

Comment:
@yisitu Do we start maintaining compatibility now (v0.0.1)? If so, then `visible_device_count` should be marked for future deprecation, and `get_device_count` should be added at the end, right?

Reply:
You *must not* delete a field from parameter struct when avoidable!  Penporn is correct, this should be marked for future deprecation instead.

So let's try to avoid deprecation as much as possible. Can we leave the field behind, but return a zero value.

Reply:
Chatted with @yisitu offline and we thought this is still an early development period and it should be okay to break compatibility. Will leave the decision to @tensorflow/api-owners